crates/hyperqueue/src/client/output/cli.rs

```diff
@@ -536,49 +541,52 @@ impl Output for CliOutput {
         );
     }
 
-    fn print_job_output(&self, job: JobDetail, task_selector: &Selector, output_stream: &OutputStream) -> anyhow::Result<()> {
+    fn print_job_output(
+        &self,
+        job: JobDetail,
+        task_selector: &Selector,
+        output_stream: &OutputStream,
+    ) -> anyhow::Result<()> {
         fn read_stream(task_state: &JobTaskState, output_stream: &OutputStream) {
             let (_, stdout, stderr) = get_task_paths(task_state);
             let stdin = match output_stream {
-                OutputStream::Stdout => {stdout}
-                OutputStream::Stderr => {stderr}
+                OutputStream::Stdout => stdout,
+                OutputStream::Stderr => stderr,
             };
 
-            let file = File::open(stdin).expect("Unable to open file");
-            let mut reader = BufReader::new(file);
-            for line in reader.lines() {
-                let line = line.expect("Unable to read line");
-                println!("{}", line);
-            }
+            match File::open(stdin) {
+                Ok(mut file) => {
+                    std::io::copy(&mut file, &mut std::io::stdout());
+                }
+                Err(_) => log::warn!("File: {} cannot be opened", stdin),
```

Comment:
```suggestion
                Err(error) => log::warn!("File {} cannot be opened: {:?}", stdin, error),
```
Let's give more context to the user.